internal/testlib/string.go

```diff
@@ -0,0 +1,13 @@
+package testlib
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// EqualIgnoreCRLF requires equality ignoring line feed differences.
+func EqualIgnoreCRLF(t *testing.T, expected, actual string) {
```

Comment:
Instead of this I would add a `.gitattributes` like https://github.com/goreleaser/nfpm/blob/master/.gitattributes